src/Helpers/Constants.cs

```diff
@@ -1,12 +1,173 @@
+    public static readonly string NBXPLORER_URI;
+    public static readonly string? NBXPLORER_BTCRPCURL;
+    public static readonly string ALICE_HOST;
+    public static readonly string CAROL_HOST;
+    public static readonly string? FUNDSMANAGER_ENDPOINT;
+    public static readonly string? COINGECKO_ENDPOINT;
+    public static readonly string? MEMPOOL_ENDPOINT;
+    public static readonly string? REMOTE_SIGNER_ENDPOINT;
+
+    // Credentials
+    public static readonly string? NBXPLORER_BTCRPCUSER;
+    public static readonly string? NBXPLORER_BTCRPCPASSWORD;
+    public static readonly string? COINGECKO_KEY;
+    public static readonly string PUSH_NOTIFICATIONS_ONESIGNAL_APP_ID;
+    public static readonly string? PUSH_NOTIFICATIONS_ONESIGNAL_API_BASE_PATH;
+    public static readonly string? PUSH_NOTIFICATIONS_ONESIGNAL_API_TOKEN;
+    public static readonly string? AWS_REGION;
+    public static readonly string? AWS_ACCESS_KEY_ID;
+    public static readonly string? AWS_SECRET_ACCESS_KEY;
+    public static readonly string? AWS_KMS_KEY_ID;
+
+    // Crons & Jobs
+    public static readonly string MONITOR_WITHDRAWALS_CRON = "10 0/5 * * * ?";
+    public static readonly string? JOB_RETRY_INTERVAL_LIST_IN_MINUTES;
+
+
+    // Observability
+    public static readonly string? OTEL_EXPORTER_ENDPOINT;
     
+    // Usage
+    public static readonly string? BITCOIN_NETWORK; 
+    public static readonly long MINIMUM_CHANNEL_CAPACITY_SATS = 20_000;
+    public static readonly decimal MINIMUM_WITHDRAWAL_BTC_AMOUNT = 0.0m;
+    public static readonly decimal MAXIMUM_WITHDRAWAL_BTC_AMOUNT = 21_000_000;
+    public static readonly int TRANSACTION_CONFIRMATION_MINIMUM_BLOCKS;
+    public static readonly long ANCHOR_CLOSINGS_MINIMUM_SATS;
+    public static readonly string DEFAULT_DERIVATION_PATH;
+    public static readonly int SESSION_TIMEOUT_MILLISECONDS = 3_600_000;
+
+
     static Constants()
     {
-        var environmentVariable = Environment.GetEnvironmentVariable("MINIMUM_CHANNEL_CAPACITY_SATS") ?? throw new InvalidOperationException("MINIMUM_CHANNEL_CAPACITY_SATS not set");
-        MINIMUM_CHANNEL_CAPACITY_SATS = long.Parse(environmentVariable, NumberStyles.AllowDecimalPoint, CultureInfo.InvariantCulture);
+        // If it is a command from ef, ignore the empty env variables
+        if (Assembly.GetEntryAssembly()?.GetName().Name?.ToLowerInvariant() == "ef") return;
+
+        IS_DEV_ENVIRONMENT = Environment.GetEnvironmentVariable("IS_DEV_ENVIRONMENT") != null;
+        // Features
+        ENABLE_REMOTE_SIGNER = Environment.GetEnvironmentVariable("ENABLE_REMOTE_SIGNER") != null;
+
+        PUSH_NOTIFICATIONS_ONESIGNAL_ENABLED = Environment.GetEnvironmentVariable("PUSH_NOTIFICATIONS_ONESIGNAL_ENABLED") != null;
+
+        ENABLE_HW_SUPPORT = Environment.GetEnvironmentVariable("ENABLE_HW_SUPPORT") != "false"; // We default to true
```

Comment:
is this ok? I copied like it was in the original file

Reply:
It looks good to me